fastlane/Fastfile

```diff
@@ -344,12 +345,94 @@ lane :build_test_app_and_frameworks do
   )
 end
 
+lane :xcmetrics do |options|
+  next unless is_check_required(sources: sources_matrix[:xcmetrics], force_check: @force_check)
+
+  ['test_output/', 'performance/'].each { |dir| FileUtils.remove_dir(dir, force: true) }
+
+  scan(
+    project: xcode_project,
+    scheme: 'StreamChatUITestsApp',
+    testplan: 'Performance',
+    result_bundle: true,
+    derived_data_path: derived_data_path,
+    cloned_source_packages_path: source_packages_path,
+    clean: is_localhost,
+    xcargs: buildcache_xcargs,
+    prelaunch_simulator: true
+  )
+
+  sh('xcparse logs test_output/StreamChatUITestsApp.xcresult test_output/')
+  action_log = File.read(Dir.glob('test_output/**/action.txt').first)
+  branch_performance = xcmetrics_log_parser(log: action_log)
+
+  sh("git clone git@github.com:GetStream/stream-swift-performance-benchmarks.git #{File.dirname(performance_path)}")
+  previous_release_branch = "release/#{last_git_tag}"
+  performance_benchmarks = JSON.parse(File.read(performance_path))
+  develop_performance = performance_benchmarks['develop']
+  previous_release_performance = performance_benchmarks[previous_release_branch]
+
+  github_comment = "## Performance Metrics\n| `metric` | `#{current_branch}` | `#{previous_release_branch}` | `develop` | `status` |\n| - | - | - | - | - |\n"
+  [:startup_time, :message_list_load_time, :channel_list_load_time, :message_list_scroll_time, :channel_list_scroll_time].each do |metric|
+    release_value = previous_release_performance[metric.to_s]
+    develop_value = develop_performance[metric.to_s]
+    branch_value = branch_performance[metric]
+
+    status = if branch_value > develop_value && branch_value < develop_value * 1.5
+               '🟡' # Warning if new value is 50% slower than develop
+             elsif branch_value > develop_value
+               '🔴' # Failure if new value is more than 50% slower than develop
+             else
+               '🟢' # Success if new value is faster than or equal to develop
+             end
```

Comment:
I'm not sure I'm following this 🤔 

What is the difference between this two?
- Warning if new value is 50% slower than develop
- Failure if new value is more than 50% slower than develop

Another thing is that 50% is way too much, something half slower is not a good 😄 We should maybe put 5% as warning, and 10% as a failure. This follows the default values of XCMeasure in Xcode.

Reply:
Yes if it is random, is not useful at all, so we should remove it 👍